Domain/ProductManagement/Product.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace LenaInventoryManagementSystem.Domain.ProductManagement
+{
+    public class Product
+    {
+        private string _productName;
+        private double _price;
+        private int _quantity;
+        private const int maxItemsInStock = 150;
+        private int _maxItemsInStock;
+        public Product()
+        {
+            _maxItemsInStock = maxItemsInStock;
+        }
+        public Product(string name) 
+        {
+            _productName = name;
+        }
+        public Product(string name, double price) : this (name)
+        {
+            _price = price;
+        }
+        public Product(string name, int quantity) : this(name)
+        {
+            _quantity = quantity;
+        }
+        public Product(string name, double price, int quantity) : this (name, price) 
+        {
+            _quantity = quantity;
+        }
+        public string ProductName
+        {
+            get => _productName;
+            set => _productName = value;
+        }
+        public double Price
+        {
+            get => _price;
+            set => _price = value;
+        }
+        public int Quantity
+        {
+            get => _quantity;
```

Comment:
these are good setters and getters, for next MRs no need to add them if we don't have special logic around them. If they just get and set the values and are public, then this doesn't add any value ;)